python/makeMultipleFileListBunches.py

```diff
@@ -38,8 +38,8 @@ def getListOfDirectories(path):
 
 
 # TODO: Oh ffs, read that from scenario config, NOT hardcode!
-filename_prefix = "Lumi_TrksQA_"
-# filename_prefix = "Koala_comp_"
```

Comment:
we shouldn't hardcode this at all, there needs to be a reference to the config here